vercel.json

```
@@ -10,6 +14,12 @@
 			"destination": "/api/404.html"
 		}
 	],
+	"redirects": [
+		{
+			"source": "/docs/ja/:path*",
+			"destination": "/docs/:path*?wovn=ja"
```

```suggestion
			"destination": "/:path*?wovn=ja"
```

I noticed that all the en redirects above strip `/docs`, should this do the same?

---

The others were rewrites, so it'll rewrite the url path. This is a redirect, so without the /docs, it'll actually be sent to braze.com/ja?wovn=ja , which would be the main marketing site instead of braze.com/docs/...?wovn=ja